drivers/misc/nxp_inputmux/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config NXP_INPUTMUX
+	bool "NXP INPUTMUX driver"
+	depends on DT_HAS_NXP_INPUTMUX_ENABLED
+	help
+	  Enables a driver for the NXP INPUTMUX peripheral.
+	  It is useful for routing internal signals in supported NXP microcontrollers.
+	default y
```

Comment:
nit but can you put the default above the help, this is just an unusual appearance, preferably put the default before the depends on as well